src/RobotInterface/YarpImplementation/src/YarpSensorBridge.cpp

```diff
@@ -106,13 +94,12 @@ bool YarpSensorBridge::setDriversList(const yarp::dev::PolyDriverList& deviceDri
                   << std ::endl;
         return false;
     }
-
+        
```

Comment:
White spaces

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/182/commits/d4dcb8a28a7f4b4a59fee3f7b96e2f25f6a4d268.